### PR TITLE
Fix multiple random number outputs

### PR DESCRIPTION
**Pull request by mahrossi**
@venkatkapil24 have a look.

### CODE REVIEW

File: ipi/inputs/simulation.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ class InputSimulation(Input):
             InputRandom,
             {
                 "help": InputRandom.default_help,
-                "default": input_default(factory=Random),
+                "default": input_default(factory=(lambda: Random())),
```
Comment (ceriottm) [inline ipi/inputs/simulation.py:60]: I don't think this is necessary if you don't hardcode a default seed
```suggestion
"default": input_default(factory=Random),
```

File: ipi/engine/simulation.py [hunk 2]
```diff
@@ -161,6 +162,12 @@ def __init__(
         self.syslist = syslist
         for s in syslist:
             s.prng = self.prng  # bind the system's prng to self prng
+            if threading.current_thread() == threading.main_thread():
+                info(
+                    "@ RANDOM SEED: The seed used in this calculation was "
+                    + str(self.prng.seed),
+                    verbosity.low,
+                )
```
Comment (ceriottm) [inline ipi/engine/simulation.py:170]: nice!

**Review by ceriottm — CHANGES_REQUESTED** [review]

File: ipi/inputs/simulation.py (hunk 1 above)
Comment (mahrossi) [reply ipi/inputs/simulation.py:60]: I forgot to erase the lambda:

**Review by mahrossi — COMMENTED** [review]
done and ready to merge

**Pull request merged**